environments/production/inventory.yml

```diff
@@ -2,6 +2,31 @@
 
 all:
   vars:
+    # Prometheus vars
     prometheus_version: 2.22.0
+
+    prometheus_web_listen_address: '0.0.0.0:9090'
```

Comment:
This would make Prometheus reachable over all interfaces. I would much prfer this on 127.0.0.1

Reply:
Done